plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/gettingstarted/GettingStartedAuthUtils.kt

```diff
@@ -229,13 +234,26 @@ fun reauthenticateWithQ(project: Project) {
 }
 
 fun emitUserState(project: Project) {
-    AuthTelemetry.userState(
-        project,
-        source = getStartupState().toString(),
-        authEnabledConnections = getEnabledConnections(project),
-        authStatus = getAuthStatus(project),
-        passive = true
-    )
+    val explorerConnection = ToolkitConnectionManager.getInstance(project).activeConnection()
```

Comment:
Right now, I am just grabbing the connection being used by the explorer. I see that the activeConnection() function is marked as deprecated, but there is no alternative for getting the explorer connection from what I've seen. The getActiveConnectionForFeature function seems to work only for Q, CodeCatalyst, and CodeWhisperer.

Reply:
If we want to look at all the active connections, I think it would be easy enough to use a function like:
https://github.com/aws/aws-toolkit-jetbrains/blob/80abea5c6be906bfa767400e5dcb19fdd2d9863a/plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedTelemetryUtils.kt#L23
to get the scopes from each active connection. If we are looking at enabled connections in this Telemetry metric, why not the scopes for those enabled connections?